ui/apps/platform/src/Containers/Risk/KeyValuePairs.js

```diff
@@ -72,6 +75,8 @@ class KeyValuePairs extends Component {
                     <span className={`flex-1 min-w-0 ${className}`}>
                         {isObject(value) || isArray(value) ? (
                             <div className="ml-2">{this.getNestedValue(value)}</div>
+                        ) : label === 'Deployment ID' ? (
+                            <Link to={`${vulnManagementPath}/deployment/${value}`}>{value}</Link>
                         ) : (
```

Comment:
I think the linter is right, nested tertiary operator is 🤮. I'd match rather do pattern matching here, e.g., via `switch(true)`:

```
switch(true) {
  case isObject(value) || isArray(value):
    <div className="ml-2">{this.getNestedValue(value)}</div>
    break
  case label === 'Deployment ID':
    <Link to={`${vulnManagementPath}/deployment/${value}`}>{value}</Link>
    break
  default:
    value.toString()
}
```

Not sure if there are syntax errors in the snippet above but it illustrates the idea.

Reply:
Noted, but I find the switch to be more brittle, ever since the infamous [GOTO FAIL vulnerability](https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2.html). The UI team is considering removing that lint rule, because we override it consistently for the pattern

Loading...
or
Error
or 
Display Data

on CRUD pages.

Reply:
Unfortunately `switch` is only a statement in JavaScript and will not return a value. Native pattern matching is the number one thing I wish would get added to JS but it remains only at [Stage 1](https://github.com/tc39/proposal-pattern-matching) after _7 years_. 😢

Reply:
Got it. Consistency is important, so if it's a common pattern in our code, I retract my comment.